packages/react-i18n/docs/react_i18n_schema.md

```diff
@@ -105,48 +120,116 @@ In order to avoid key collisions, the following keys are reserved for use as plu
 
 **Example INVALID English file:**
 ```jsonc
 {
   "foo": "bar",
-  // 🚫 INVALID: The use of `other` makes this a pluralization context,
+  // 🚫 INVALID:
+  // The use of `other` key outside of an ordinal pluralization context
+  // makes this a cardinal pluralization context,
   // but it is missing the required `one` sibling key that English needs
-  // therefore it is not a valid pluralization context.
+  // therefore it is not a valid cardinal pluralization context.
   // It is either:
-  //   * not intended to be a pluralization context (in which case the reserved `other` key needs to be renamed)
-  //   * missing the `one` sibling key (in which case the `one` key needs to be added)
+  //   * not intended to be a cardinal pluralization context (in which case the reserved `other` key needs to be renamed)
+  //   * missing the required `one` sibling cardinal pluralization key (in which case the `one` key needs to be added)
+  //   * meant to be part of an ordinal pluralization context (in which case it needs to be placed under an "ordinal"
+  //     key and the other sibling ordinal pluralization keys need to be added)
   "other": "Other"
 }
 ```
 
-## Invalid pluralization keys
+### Invalid cardinal pluralization keys
 
-Furthermore, pluralization keys that are not required by the language must not be used, even within a pluralization context.
+Furthermore, cardinal pluralization keys that are not required by the language must not be used, even within a cardinal pluralization context.
 
-For example, even though `zero` is a required pluralization key for some languages (eg. Arabic), it is not a required pluralization key in English. Therefore, the `zero` key must not be present in an English file alongside the pluralization KVPs required by English.
+For example, even though `zero` is a required cardinal pluralization key for some languages (eg. Arabic), it is not a required cardinal pluralization key in English. Therefore, the `zero` key must not be present in an English file alongside the cardinal pluralization KVPs required by English.
 
 **Example INVALID English file:**
 ```jsonc
 {
   "cars": {
-    "zero": "I have no cars", // 🚫 INVALID: `zero` is not a required pluralization key for English!
+    "zero": "I have no cars", // 🚫 INVALID: `zero` is not a required cardinal pluralization key for English!
     "one": "I have {count} car",
     "other": "I have {count} cars"
   }
 }
 ```
 
-## Detecting pluralization contexts
+### Detecting cardinal pluralization contexts
+
+These constraints allow for a simple algorithm for identifying cardinal pluralization contexts:
+
+* If a node has child leaf KVPs for **any** of the cardinal pluralization keys required by the language, outside of a then its **immediate** children are within a cardinal pluralization context.
```

Comment:
`outside of a then its` doesn't make sense to me, I think you're missing a word here.

Reply:
Very true. Fixed.